atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -198,56 +200,66 @@ def run(self):
+            if base_image is not None and not self.workflow.builder.base_from_scratch:
+                base_image_name = base_image.to_str()
+                try:
+                    base_image_id = self.workflow.builder.base_image_inspect['Id']
+                except KeyError:
+                    base_image_id = ""
+            else:
+                base_image_name = ""
                 base_image_id = ""
-        else:
-            base_image_name = ""
-            base_image_id = ""
 
-        try:
-            with open(self.workflow.builder.df_path) as f:
-                dockerfile_contents = f.read()
-        except AttributeError:
-            dockerfile_contents = ""
+            try:
+                with open(self.workflow.builder.df_path) as f:
+                    dockerfile_contents = f.read()
+            except AttributeError:
+                dockerfile_contents = ""
 
-        parent_images_strings = self.workflow.builder.parent_images_to_str()
-        if self.workflow.builder.base_from_scratch:
-            parent_images_strings[SCRATCH_FROM] = SCRATCH_FROM
+            parent_images_strings = self.workflow.builder.parent_images_to_str()
+            if self.workflow.builder.base_from_scratch:
+                parent_images_strings[SCRATCH_FROM] = SCRATCH_FROM
 
         annotations = {
-            "dockerfile": dockerfile_contents,
-            "repositories": json.dumps(self.get_repositories()),
-            "commit_id": commit_id,
-            "base-image-id": base_image_id,
-            "base-image-name": base_image_name,
-            "image-id": self.workflow.builder.image_id or '',
-            "digests": json.dumps(self.get_pullspecs(self.get_digests())),
-            "parent_images": json.dumps(parent_images_strings),
-            "plugins-metadata": json.dumps(self.get_plugin_metadata()),
-            "filesystem": json.dumps(self.get_filesystem_metadata()),
+            'repositories': json.dumps(self.get_repositories()),
+            'digests': json.dumps(self.get_pullspecs(self.get_digests())),
+            'plugins-metadata': json.dumps(self.get_plugin_metadata()),
+            'filesystem': json.dumps(self.get_filesystem_metadata()),
         }
 
-        help_result = self.workflow.prebuild_results.get(AddHelpPlugin.key)
-        if isinstance(help_result, dict) and 'help_file' in help_result and 'status' in help_result:
-            if help_result['status'] == AddHelpPlugin.NO_HELP_FILE_FOUND:
-                annotations['help_file'] = json.dumps(None)
-            elif help_result['status'] == AddHelpPlugin.HELP_GENERATED:
-                annotations['help_file'] = json.dumps(help_result['help_file'])
-            else:
-                self.log.error("Unknown result from add_help plugin: %s", help_result)
+        if self.source_build:
+            source_result = self.workflow.prebuild_results[PLUGIN_FETCH_SOURCES_KEY]
+            annotations['sources_for_nvr'] = source_result['sources_for_nvr']
```

Comment:
Minor: if this was a label instead we'd be able to do: `oc get builds -l sources_for_nvr=...` to easily find builds for a given nvr.

Reply:
ack will do